### PR TITLE
Add test with JEST 

### PR DESCRIPTION
**Pull request by lmunozm1702**
This PR includes jest testing over Calculator components.

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi Team
I'm sure you put a lot of effort into this project but this PR is invalid because you have failed linters checks😓 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lmunozm1702** [comment]
The bug in the linter was not in the code, but in the github linter setup, we restarted the task and it finished successfully. We request a new code review.

image

File: src/__tests__/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
```
Comment (gemmen29) [inline src/__tests__/Calculator.test.js:2]: - [x] Kindly add tests that simulate user interaction as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20React%20Testing%20Library%20to%20simulate%20user%20interaction.).

File: src/__tests__/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// import renderer from 'react-test-renderer';
+import calculate from '../logic/calculate';
```
Comment (gemmen29) [inline src/__tests__/calculate.test.js:2]: - [ ] Kindly test all functionality and scenarios for the `calculate` file as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Write%20unit%20tests%20for%20the%20files%20operate.js%20and%20calculate.js%20using%20Jest.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/calculate.test.js (hunk 2 above)
Comment (lmunozm1702) [reply src/__tests__/calculate.test.js:2]: Hi! it's done!

Comment (gemmen29) [reply src/__tests__/calculate.test.js:2]: Hi @lmunozm1702 you only test the `AC` operation. You must test all other operations and scenarios.

Comment (lmunozm1702) [reply src/__tests__/calculate.test.js:2]: I'm sorry!!! I write the comment but forget the commit and push, now is really done!

Comment (gemmen29) [reply src/__tests__/calculate.test.js:2]: Ok, No problem.

File: src/__tests__/Calculator.test.js (hunk 1 above)
Comment (lmunozm1702) [reply src/__tests__/Calculator.test.js:2]: It's done, we add a test simulating the pressed key "8" and then the key "7" and find the value "87" on the screen.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @lmunozm1702 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**